monai/inferers/utils.py

```diff
@@ -300,6 +300,7 @@ def sliding_window_inference(
 
     # remove padding if image_size smaller than roi_size
     if any(pad_size):
+        kwargs.update({'pad_size': pad_size})
```

Comment:
Hi @heyufan1995 , is it necessary to add this line? I checked the vista3d bundle and it does not have this line